Exp fqhc diagnostic compliance comparison

korourke5 opened the pull request:
# CRCsim PR Template

## Description
<!-- Provide a brief description of the changes in this PR -->

## PR Type
<!-- Mark the appropriate option with an [x] (no spaces around the x) -->
- [ ] Feature (intended to merge)
- [X] Experiment (not intended to merge)

<!-- If this is an experiment PR, delete the following section and complete the experiment-specific sections of this template. -->
<!-- If this is a feature PR, complete the following section and delete the experiment-specific sections. -->

## Experiment review checklist
<!-- Make sure all items are checked before submitting the PR -->
- [X] This experiment branch has the `exp-` prefix
- [X] This experiment uses the appropriate `crcsim` commit hash (typically, the latest commit in the `main` branch). This ensures that the experiment uses the latest version of the model.
- [X] This experiment uses the correct baseline `parameters.json` file
    - [X] Correct screening start and end ages for the experiment
    - [X] Latest values of all calibrated parameters
    - [X] All necessary tests and latest test parameters
    - [X] Latest cost parameters
- [ ] My `prepare.py` script applies incidence rate ratio (IRR) adjustment if appropriate for this experiment - NOT APPROPRIATE
- [X] I have run `prepare.py` and:
    - [X] Confirmed that all scenarios necessary for this experiment were created
    - [X] Confirmed that no extraneous scenarios are in the `scenarios/` directory (eg, uncommitted local changes from a previous experiment)
    - [X] Spot-checked a sample of scenario parameter files to ensure that my scenario creation logic works as expected
- [X] I have updated `crcsim/experiment/README.md` with detailed information about the experiment's goals, scenarios, and corresponding AWS objects
- [X] I am opening this PR as a **Draft** PR

## Experiment review process
<!-- No checkboxes here, because this happens after you submit the PR -->
*All experiments must follow each step of this review process.*
1. Contributor checks all items in experiment review checklist
1. Contributor opens **Draft PR** and tags a collaborator to review
1. Reviewer conducts a thorough review, including pulling experiment branch, running `prepare.py`, and spot-checking scenarios
1. Contributor and reviewer address any issues identified during review
1. Reviewer explicitly approves PR
1. Contributor builds image and pushes to ECR, copies experiment files to S3, and runs experiment
1. Contributor analyzes experiment (eg `crcsim/experiment/summarize.py`)
1. Contributor pushes results summary, so we have documentation of detailed results (eg `crcsim/experiment/summary/summarized.xlsx`)
1. Contributor updates experiment README with a detailed summary of results
1. Contributor closes this PR with an informative comment (eg, brief summary of results)

**Note that experiment PRs are never merged into `main`!** The PR is closed, and the experiment is maintained as a separate branch. That's why we keep all experiment PRs as drafts.

korourke5 committed:
Update prepare.py for different FQHC scenarios and different diagnostic compliance scenarios.

korourke5 committed:
Update parameters.json with effectiveness and cost estimates.

korourke5 committed:
Update run_iteration.sh for AWS S3

korourke5 committed:
Update README.md with instructions for this experiment

korourke5 committed:
Update prepare.py to shorter scenario names.

korourke5 committed:
Update prepare.py so all scenario names are
properly created.

korourke5 committed:
Updated scenarios.

sandypreiss reviewed (commented):
This is ready to run. 

But note that you'll need to change the S3 bucket constant in `summarize.py` before you summarize the experiment.

I added some stylistic comments to `prepare.py`. They won't affect the scenarios, so no need to resolve before running. But please do resolve them before we close the PR. (Now that I think about it, I'm not sure how the missing type hints got by the linter - I'll have to look into that.)

korourke5 committed:
Updated files to include updated AWS Batch.
Changed scenario names so AWS would accept them
as job names. Including scenarios and summary
files.

korourke5 committed:
Added information to the ReadMe file about results

korourke5 committed:
Edited prepare.py with suggested changes.

sandypreiss commented:
This experiment was a sense check to make sure outcomes under a given diagnostic compliance rate matched those of a previous run with same parameters. They did.
